acceptance/config/git/options.rb

```diff
@@ -3,7 +3,7 @@
   :install                     => [
     'puppet',
   ],
-  'is_puppetserver'            => true,
+  'is_puppetserver'            => false,
```

Comment:
@joshcooper IMO this is the most controversial change in this PR. If we don't have this, the setup will complain about needing at least one host with the label master